railties/lib/rails/commands/test/test_command.rb

```diff
@@ -1,21 +1,41 @@
 require "rails/command"
-require "rails/test_unit/minitest_plugin"
+require "rails/test_unit/runner"
 
 module Rails
   module Command
     class TestCommand < Base # :nodoc:
       no_commands do
         def help
```

Comment:
The test runner of the rails plugin does not use `TestCommand`.
Therefore, `TestCommand#help` will not be called and rails' help will not be displayed. 

``` 
$ rails plugin new yaffle  
$ cd yaffle 
$ <edit some files> 
$ ./bin/test --help
minitest options:
    -h, --help                       Display this help.
    -s, --seed SEED                  Sets random seed. Also via env. Eg: SEED=n rake
    -v, --verbose                    Verbose. Show progress processing files.
    -n, --name PATTERN               Filter run on /regexp/ or string.
    -e, --exclude PATTERN            Exclude /regexp/ or string from run.

Known extensions: rails, pride
    -b, --backtrace                  Show the complete backtrace
    -d, --defer-output               Output test failures and errors after the test run
    -f, --fail-fast                  Abort test run on first failure or error
    -c, --[no-]color                 Enable color in the output
    -p, --pride                      Pride. Show your testing pride!
``` 

I think that it is better to keep help displayed on `plugin_rails_options`, what about it?